python/cudf/cudf/_fuzz_testing/csv.py

```diff
@@ -70,10 +71,10 @@ def write_data(self, file_name):
         if self._current_buffer is not None:
             self._current_buffer.to_csv(file_name + "_crash.csv")
 
-    def get_rand_params(self, params):
+    def set_rand_params(self, params):
         params_dict = {}
         for param, values in params.items():
-            if param == "usecols" and values is None:
+            if param == "usecols" and values == ALL_POSSIBLE_VALUES:
```

Comment:
It might be less checking to just check if `values == ALL_POSSIBLE_VALUES` and then look for what to do based on the param name after

Reply:
Done, addressed these changes